test/LogLevelConfigurationTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace AbacaphiliacTest\Doctrine;
+
+use Abacaphiliac\Doctrine\LogLevelConfiguration;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LogLevel;
+
+/**
+ * @covers \Abacaphiliac\Doctrine\LogLevelConfiguration
+ */
+class LogLevelConfigurationTest extends TestCase
+{
+    /** @var LogLevelConfiguration */
+    private $sut;
+
+    protected function setUp()
+    {
+        $this->sut = new LogLevelConfiguration([
+            LogLevel::DEBUG => 0,
+            LogLevel::INFO => 10,
+            LogLevel::NOTICE => 20,
+            LogLevel::WARNING => 30,
+            LogLevel::ERROR => 40,
+            LogLevel::CRITICAL => 50,
+            LogLevel::ALERT => 60,
+            LogLevel::EMERGENCY => 70,
+        ]);
+    }
+
+    public static function dataValidLevels(): array
+    {
+        return [
+            [-1, null],
+            [0, null],
```

Comment:
@holtkamp this case in particular was a surprise to me. i will add a branch to the log level config object to address this case (i feel it should have returned `debug`), and another test to explicitly cover the lower bound, outside of the data provider.

Reply:
In case no match is found, the default level should be returned...?

Reply:
oh, debug is set to exactly 0 in the setup of this test. hence the branch to check the slim chance of an exact match in #17 

in the case where no match is found i believe null is correct. are you thinking that should be changed as well?

Reply:
No, I think for this class `NULL` is correct indeed. I did not look in detail. The Logger itself then fallback to the default log level. 

Nice, you are very thorough in the testing part, I can learn a lot from that stance!